### ISSUE TITLE
Invalid residues

### ISSUE THREAD

**Issue by skodapetr** [skodapetr/p2rank-funpdbe#2]
For some PDB files, we have residues that are not part of the residues as defined in the pdbe.

For example 1OSS. The residues can be seen at the PDBe [site](https://www.ebi.ac.uk/pdbe/api/pdb/entry/residue_listing/1oss/chain/A) that is used by the validator.
Our input data contains residue ```177 ALA``` where according to validation it should probably be ```177 PHE```.

There is ```150 ALA``` with the ```author_residue_number``` equal to ```177```.

**Comment by rdk** [comment]
1OSS contains 2 residues with seq.number 177:

PDB file:
```
1. A_177 (VAL)
2. A_177A (ALA)
```
In PDBe residue_listing *author_residue_number* and *author_insertion_code* code is taken from original PDB file and *residue_number* is their own arbitrary renumbering of residues.

https://www.ebi.ac.uk/pdbe/api/pdb/entry/residue_listing/1oss/chain/A:
```
1. {"author_residue_number":177, "author_insertion_code":"", "observed_ratio":1, "residue_name":"VAL","residue_number":150}
2. {"author_residue_number":177, "author_insertion_code":"A","observed_ratio":1, "residue_name":"ALA","residue_number":151},
```

The question is which residue numbering is expected by funpdbe, (a) the one from original PDB file or (b) the one from their residue_listing . I guess (a) and that the validation failed because of the bug #1 .

**Comment by skodapetr** [comment]
On the test data this problem no longer appearch after #1 was fixed.
